content/plugin/sdkv2/sdkv2-intro.mdx

```diff
@@ -23,3 +23,15 @@ Terraform Plugin SDKv2 is an established way to develop Terraform Plugins. It is
 
 - Learn how to [debug your provider](/plugin/sdkv2/debugging) using either logging calls or a debugging tool.
 - Learn how to [write successful acceptance and unit tests](/plugin/sdkv2/testing) for your provider.
+
+## Combine or Translate
+
+* [Combine your provider](/plugin/mux/combine-protocol-version-5-providers) with other [protocol version 5](/plugin/how-terraform-works#protocol-version-5) providers.
+* [Translate your provider](/plugin/mux/translate-protocol-version-5-to-6) into a [protocol version 6](/plugin/how-terraform-works#protocol-version-6) provider to require Terraform 1.0 and later.
+
+## Migrate to Terraform Plugin Framework
+
+The [terraform-plugin-framework](/plugin/framework) is a new way to develop Terraform providers, offering improvements and new features from Terraform Plugin SDKv2. You can refactor individual resources and data sources over time with the following compatibility:
+
+* Terraform 0.12 and later: First [translate your provider](/plugin/mux/translate-protocol-version-6-to-5) into a protocol version 5 provider. Then [combine your provider](/plugin/mux/combine-protocol-version-5-providers) with the translated provider. You will not be able to use [Protocol version 6](/plugin/how-terraform-works#protocol-version-6) features.
```

Comment:
The TINIEST nit. we don't capitalize "Protocol" elsewhere on this page. Should it be? I'm wondering now if we're inconsistent about that

```suggestion
* Terraform 0.12 and later: First [translate your provider](/plugin/mux/translate-protocol-version-6-to-5) into a protocol version 5 provider. Then [combine your provider](/plugin/mux/combine-protocol-version-5-providers) with the translated provider. You will not be able to use [protocol version 6](/plugin/how-terraform-works#protocol-version-6) features.
```

Reply:
No problem, will do a pass to ensure they are all lowercase.